### PR TITLE
[OBSDEF-15209][OBSDEF-13968] wrapper for progress bar and modal related changes

### PR DESCRIPTION
**Pull request by tejasjc-ecs**
Summary of changes:

**Progress bars:**

- Changed colors of progress bar as per DDS
- Positioned label correctly 
- Styles for static, looped progress bar
- Added progress bar stories

**Modals:**

-  ~~Changed modals size as per DDS~~ (reverted this changes as it was braking modals in standalone UI)
- Added styles for title icons
- Updated modals stories as per type

~~[Note: PR validation will run successfully once [#240](https://github.com/EMCECS/clarity-react/pull/240) is merged]~~ (Passed)
Progress bar:

stbprog

Modals;

stbicon

### CODE REVIEW

File: src/components/progressBar/ProgressBar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    ProgressBarAnimation
+} from "@dellstorage/clarity-react/progress_bars/ProgressBars";
+
+storiesOf("ProgressBar", module)
+    .add("ProgressBar Simple", () => (
+        <div>
+            <h4> Progress Bar </h4>
+            <div style={{width: "80%"}}>
+                <ProgressBar value={10} max={100} style={{width: "50%"}}>
+                    {" "}
```
Comment (akate1) [inline src/components/progressBar/ProgressBar.stories.tsx:27]: Please remove {""} from all files

**Review by akate1 — COMMENTED** [review]
@tejasjc-ecs Please update version of clarity-react once https://github.com/EMCECS/clarity-react/pull/240 is merged. 
Please update version of this package as well

File: src/components/progressBar/ProgressBar.stories.tsx (hunk 1 above)
Comment (tejasjc-ecs) [reply src/components/progressBar/ProgressBar.stories.tsx]: Done

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dellstorage/dell-design-react-common",
   "description": "Override CSS of Clarity-React components to align it with Dell design standards",
-  "version": "0.1.4",
+  "version": "0.1.5",
```
Comment (akate1) [inline package.json:4]: Please update version of clarity react lib as well

Comment (tejasjc-ecs) [reply package.json:4]: Version of clarity react was already set to 1.1.13 for this repo hence didn't update it and yarn.lock is updated now to take 1.1.14

Comment (akate1) [reply package.json:4]: ok

**Review by akate1 — APPROVED** [review]

**Review by psivakumaran — APPROVED** [review]

**Pull request merged**